chase/frontend/components/dashboard/speakers_list.tsx

```diff
@@ -4,14 +4,9 @@ import SpeakerBlock from "@/components/speakers_list/speaker_block";
 import CommentBlock from "@/components/speakers_list/comment_block";
 import QueueBlock from "@/components/speakers_list/queue_block";
 import { CountryCode, SpeakersListData } from "@/custom_types";
-
+import { useI18nContext } from "@/i18n/i18n-react";
 import "./markdown.scss";
-
-interface Speaker {
-  countryCode: CountryCode;
-  customName?: string;
-  time: string;
-}
+import { AnimatePresence, motion } from "framer-motion";
```

Comment:
And I found this very extensive and useful second library [Framer Motion](https://www.framer.com/motion/) for all animations in need of more customization

Reply:
Heard a lot of good things about it from @TheXires so I think thats a good decision. If you need support with it, best would be to contact robin